feriado/halloween21/index.html

```diff
@@ -9835,7 +9855,7 @@ <h3 class="desc">
                         </div>
                         <div class="text-preheader">
                             <p>Elas estão esperando você!</p>
-                            <a href="https://beta.meupatrocinio.com/cadastro?utm_campaign=halloween-3days-premium-free-sd&utm_source=mp-mktg" target="_blank" class="btn btn-primary btn-sign-in p-3 w-100">Vá para o site</a>
+                            <a href="https://out.meupatrocinio.com.br/51531bf6-0271-422f-a9e7-0554d68a3b2a" target="_blank" class="btn btn-primary btn-sign-in p-3 w-100">Vá para o site</a>
```

Comment:
you need to change the rest of the /click urls on this page as well

Reply:
Already changed all /click links